crates/proto/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "trust-dns-proto"
-version = "0.22.0"
+version = "0.23.0-alpha.1"
```

Comment:
This version number can also be referenced from the workspace just like the version number of dependencies.

Wouldn't it be better to define the version number in the workspace and reference it here?

Reply:
https://betterprogramming.pub/workspace-inheritance-in-rust-65d0bb8f9424

Reply:
ah, cool, I missed that. I'm going to move all common configs over to that. thanks for pointing that out.